en/services/opendroneid.md

```diff
@@ -23,13 +30,13 @@ The broadcast method used with Bluetooth Legacy Advertising signals impose a str
 Therefore the relevant data is divided into different categories and each category is transmitted via its own message.
 
 The standards defines 6 such messages and an additional 7th message type for packing multiple messages together into a message pack (used when transmitting on Wi-Fi NaN, Wi-Fi Beacon or Bluetooth Long Range with Extended Advertising).
-To support easy data transfers to/from a drone ID transmitter/receiver, MAVLink messages supporting all the fields of the drone ID messages have been made available.
+To support easy data transfers to/from a Remote ID (RID) transmitter/receiver component, MAVLink messages supporting all the fields of the drone ID messages have been made available.
 See [Messages](#messages) below.
 
 ## Use case examples {#use_cases}
 
 There are multiple possible use cases for the MAVLink drone ID messages:
```

Comment:
Which of the use cases listed below are mandatory to be supported for the RID compliance? Worth mentioning?

Reply:
The only real mandatory use case is that the UA broadcasts valid ID etc. data. How that data is distributed inside the UAS doesn't really matter from a regulatory perspective. There is no requirement to use MAVLink messages, so none of the listed use cases are mandatory.